package.json

```diff
@@ -13,9 +13,11 @@
 		"lint-staged": "pretty-quick --staged && lint-staged"
 	},
 	"dependencies": {
+		"@react-native-async-storage/async-storage": "^1.15.14",
 		"@unimodules/core": "~7.1.1",
 		"@unimodules/react-native-adapter": "~6.3.4",
 		"axios": "^0.24.0",
+		"crypto-js": "^4.1.1",
```

Comment:
why crypto-js instead of tiny-sha256? did you try tiny-sha256?

Reply:
To be honest, I couldn't figure out how to implement tiny-sha256